script.js

```diff
@@ -0,0 +1,12 @@
+function show() {
```

Comment:
- [ ] Kindly make sure that when you click on a link in the mobile menu, the menu should disappear and the page should move to the specific section. for now, clicking on the menu link doesn't do anything

![Screenshot_20230801_090003](https://github.com/abdullah97-pro/Portfolio-Mobile-Project/assets/25227022/1c64bf2f-e860-4262-b58c-d2dbf95b9d16)